lib/v2/lever/router.js

```diff
@@ -0,0 +1,3 @@
+module.exports = function (router) {
+    router.get('/lever/:domain', require('./index.js'));
```

Comment:
```suggestion
    router.get('/:domain', require('./index.js'));
```

Reply:
Thank you @TonyRL for the suggested changes - and I apologize for alerting you, I thought I was committing to a branch. I also deeply appreciate your suggestions and have tested them and found them to have fixed my route(s). 

One question I have is your comment about:

"
Please also provide routes with actual parameter(s) for testing.
"

Do you mean in other.md provide example routes that people could see an example feed? E.g. /lever/lever (as it is now)

Reply:
Put `/lever/lever` into the \`\`\`routes\`\`\` region in your PR description for automated testing